ontology/kisao.md

```diff
@@ -24,6 +29,48 @@ license:
 activity_status: active
 repository: https://github.com/SED-ML/KiSAO
 preferredPrefix: KISAO
+usages:
+  - user: https://sed-ml.org/
+    type: annotation
+    description: The Simulation Experiment Description Markup Language (SED-ML) is a language for describing simulations and visualizations of their results.
+    examples:
+      - url: https://sed-ml.org/examples.html
+        description: "Several examples of simulations encoded in SED-ML"
+  - user: https://biosimulations.org/
+    type: annotation
+    description: BioSimulations is a repository of biosimulation projects.
+    examples:
+      - url: https://biosimulations.org/projects/Repressilator-Elowitz-Nature-2000
+        description: "Simulation of a synthetic oscillatory biochemical network"
+  - user: https://run.biosimulations.org/
+    type: annotation
+    description: runBioSimulations is a web application for execution biological simulations.
+    examples:
+      - url: https://run.biosimulations.org/runs?try=1
+        description: "Example simulation runs"
+  - user: https://biosimulators.org/
+    seeAlso: https://arxiv.org/abs/2203.06732
+    type: annotation
+    description: BioSimulators is a registry of biosimulation tools.
+    examples:
+      - url: https://biosimulators.org/simulators/tellurium
```

Comment:
How can I understand from here how KISAO is used?

Reply:
These are projects that use KiSAO.

For example, BioSimulators uses KiSAO to categorize simulation tools (capture the kinds of models each tool can simulate). The specific KiSAO terms supported by each tool are presented in the "Algorithms" tab. For example, this displays the algorithms supported by the tellurium simulation tool: https://biosimulators.org/simulators/tellurium/latest#tab=algorithms.